src/utils/features_preprocessing.py

```diff
@@ -335,21 +380,24 @@ def fit_transform(self, x, y=None):
     # Split test
     train_X, train_y = Split(df=df).get_train_data()
 
-    # CustomTfidf test
-    tfidf = CustomTfidf().fit_transform(train_X)
+    # # CustomTfidf test
+    # tfidf = CustomTfidf().fit_transform(train_X)
+
+    # # AddingFeatures create test
+    # additional_features = AddingFeatures().create(input_df=train_X)
+    # print(f"additional_features: {type(additional_features)}")
 
-    # AddingFeatures create test
-    additional_features = AddingFeatures().create(input_df=train_X)
-    print(f"additional_features: {type(additional_features)}")
+    # # AddingFeatures normalize test
+    # normalize = AddingFeatures().normalize(additional_features)
+    # print(f"normalize: {type(normalize)}")
 
-    # AddingFeatures normalize test
-    normalize = AddingFeatures().normalize(additional_features)
-    print(f"normalize: {type(normalize)}")
+    # # AddingFeatures stack test
+    # stack = AddingFeatures().stack(tfidf, normalize)
+    # print(f"stack: {type(stack)}")
 
-    # AddingFeatures stack test
-    stack = AddingFeatures().stack(tfidf, normalize)
-    print(f"stack: {type(stack)}")
     # Preprocessor test
-    p = Preprocessor().fit_transform(train_X)
-    p = Preprocessor().fit(train_X).transform(train_X)
-    print(p)
+    p = Preprocessor(vectorizer_type='spacy').fit_transform(train_X)
+    # p = Preprocessor().fit(train_X).transform(train_X)
+    print(p.nlp) #  spacy.lang.en.English object at 0x00000264AEE66AC0
+    print(p.dim) #  300
+    print(p.__class__) #  class '__main__.SpacyVectorTransformer'
```

Comment:
maybe this should be test? or let's remove commented code

Reply:
test like separate file with unittest or pytest ?